examples/complicated/entities.go

```diff
@@ -30,7 +30,12 @@ type User struct {
 	Alias                          DogsMapAlias
 	AliasOnAlias                   AliasOnDogsMapAlias
 	AliasOnAliasWithCustomValidate AliasOnDogsMapAlias       `validate:"func=.ValidateAlias"`
-	MapOfMap                       map[string]map[int]string `validate:"value=[min_items=1,value=[min_len=3]]"`
+	MapOfMap                       map[string]map[int]string `validate:"key=[min_len=3],value=[min_items=1,value=[min_len=3]]"`
```

Comment:
Please, add unit tests for new cases ( entities_test.go )

Reply:
I added tests for new deep nested fields and add `go test` to `Makefile` to run them by Travis.